README.rst

```diff
@@ -57,6 +57,37 @@ can find it at `PULL_REQUEST_TEMPLATE.md <.github/PULL_REQUEST_TEMPLATE.md>`_.
 The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
 can find it at `ISSUE_TEMPLATE.md <.github/ISSUE_TEMPLATE.md>`_.
 
+Getting Started
+-----------------
+If you want to run dbt-schema-builder locally, clone this repository on your local system.
+You need a python virtual environment to run this, use the following commands to create and activate python 3.6.12 virtual environment
+::
+    $ pyenv install 3.6.12
+    $ pyenv virtualenv 3.6.12 schemabuilder-venv
+    $ pyenv activate schemabuilder-venv
+    $ cd dbt-schema-builder
+
+Next, you will run following commands
+::
+    $ pip install -U pip-tools
+    $ pip-compile --rebuild --upgrade  -o requirements/pip-tools.txt requirements/pip-tools.in
+
+Run the following to update the versions in all of the local requirements files    
+::
+    $ make upgrade
+
+The following command will install Schema Builder and dependencies installed in your virtualenv
+::
+    $ pip install .
+
+Now you will be all set to run dbt-schema-builder tool locally, cd into warehouse-transforms project directory in which you want to run dbt-schema-builder tool
+::
+    $ cd warehouse-transforms/projects/automated/applications
+
+Replace the --profiles-dir path
```

Comment:
The flags on this command will all be different for people who aren't using the edX repos, so we should generalize this example as well.